calico/felix/felix.py

```diff
@@ -242,6 +242,11 @@ def dump_top_level_actors(log):
             _log.warning("Unable to install diag dump handler")
             pass
 
+        # Signal Version
+        if config.VERSION_SIGNAL:
+            felix_version = "v1.0" # ToDo - get the real Felix version
```

Comment:
I think you need to get this from setuptools.  Something like
```
import pkg_resources  # part of setuptools
version = pkg_resources.require("calico")[0].version
```
From http://stackoverflow.com/questions/2058802/how-can-i-get-the-version-defined-in-setup-py-setuptools-in-my-package